### DISCUSSION TITLE
replay bitbucket cloud PR

### DISCUSSION THREAD

**Discussion by monwolf** [renovatebot/renovate#26138] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.51

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### What is your question?

We are having some artifact problems due to a wrong setup, so we are interested in recreating some PRs and rebuilding the branches. 

We've recently migrated from bitbucket server to cloud, in the first one we could rerun this removing the PR and the branch, but in this one PRs can't be removed just declined. 

There's some WA to rerun the PR in bitbucket cloud, I've been surfing the docs and I couldn't find it. 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
If you rename an existing PR to insert the string `rebase!` at the start, I think that should trigger it to rebase/reapply the changes on the existing branch

**Comment by monwolf** [comment]
Let me test it, thx :)

**Comment by monwolf** [comment]
It worked :)
